### PR TITLE
Add to Cart added

### PR DESCRIPTION
**Pull request by shubhwale**

### CODE REVIEW

File: src/app/cart/cart.component.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+})
+export class CartComponent implements OnInit {
+
+  public cartComponentRef = CartComponent;
+
+  @Input() static cartItems : Cart[]= [];
+  
+  totalPrice: number=0;
+
+  constructor(config: NgbRatingConfig, private cart: CartService) {
```
Comment (ketannaik) [inline src/app/cart/cart.component.ts:21]: Why is rating required in CartComponent?

File: src/app/cart/cart.component.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  totalPrice: number=0;
+
+  constructor(config: NgbRatingConfig, private cart: CartService) { 
+    config.readonly = true;
+    config.max=5;
+  }
+
+  quantityArray: number[] = [1,2,3,4,5];
+
+  async ngOnInit(): Promise<void> {
```
Comment (ketannaik) [inline src/app/cart/cart.component.ts:28]: Is async and await required for this functionality? Can it not be done without that?

File: src/app/cart/cart.component.ts (hunk 1 above)
Comment (shubhwale) [reply src/app/cart/cart.component.ts:20]: According to that talk we had today, I am keeping the rating for now

File: src/app/cart/cart.component.ts (hunk 2 above)
Comment (shubhwale) [reply src/app/cart/cart.component.ts]: changes done

**Pull request merged**